### DISCUSSION TITLE
百度网盘下载 视频文件外的文件 速度很慢！！！

### DISCUSSION THREAD

**Discussion by kitkw** [AlistGo/alist#1282] [General]
第一次用这个插件。在路由(openwrt)上直接docker拉取安装一切顺利成功用上。但很奇怪只有视频类文件才有速度，网页直接观看或点击下载挂载到potplayer都没问题 完全不卡。

我网盘上有一些ISO IMG RAR 下载这些却只有1-200K.......(应该不是文件大小限制，我很多视频文件都比这些大也没问题)，用IDM多线程也没用。

还有不能手动复制直连  粘贴到IDM下载，会获取不到文件的真实大小（只下载一个百K大的文件）。是不是要设置UA ？应该怎样设置！

**Comment by kitkw** [comment]
download api *  
 选择的是 crack 
official是全没速度

**Comment by anwen-anyi** [comment]
crack 本来就是视频类型的接口，其他格式的只能听天由命了 有时候快有时候没速度压根  哈哈 QAQ~~~~~~

**Reply by kitkw** [reply]
> crack 本来就是视频类型的接口，其他格式的只能听天由命了 有时候快有时候没速度压根 哈哈 QAQ~~~~~~

区分得这么细哦....... 。  顺道问一下怎样设置同步时间？？现在上传了文件  alist列表要很长时间才显示出来

**Reply by anwen-anyi** [reply]
> > crack 本来就是视频类型的接口，其他格式的只能听天由命了 有时候快有时候没速度压根 哈哈 QAQ~~~~~~
> 
> 区分得这么细哦....... 。 顺道问一下怎样设置同步时间？？现在上传了文件 alist列表要很长时间才显示出来

默认是60分钟刷新吧?你自己看 data/config.json 配置文件里面... 如果着急用你可以在你需要刷新的目录 里面右键刷新Alist,记得登录管理员，还有切记不是刷新网页哦...........

**Reply by kitkw** [reply]
> > > crack 本来就是视频类型的接口，其他格式的只能听天由命了 有时候快有时候没速度压根 哈哈 QAQ~~~~~~
> > 
> > 
> > 区分得这么细哦....... 。 顺道问一下怎样设置同步时间？？现在上传了文件 alist列表要很长时间才显示出来
> 
> 默认是60分钟刷新吧?你自己看 data/config.json 配置文件里面... 如果着急用你可以在你需要刷新的目录 里面右键刷新Alist,记得登录管理员，还有切记不是刷新网页哦...........

谢谢！在config.json改成1分钟了👍。但为什么我的alist列表不能右键（右键显示的是浏览器的右键），还一直奇怪看很多教程都有右键的管理自己的没有..............
